submissions/career/career.md

```diff
@@ -0,0 +1,53 @@
+- The introduction works great, but imo doesn't really connect to the next paragraph, but ends quite abruptly
+- Your anecdote feels a bit off, it gets a bit to far from STEM, but it is definitely nice to read and highlights some of the reasons someone might get hooked by it
+ - I feel like the CS part is a bit on the shorter side, as well as the Correlation vs Causation part. Regarding the latter, you could maybe talk a bit more about the younger audience, there are definitely people that grew up with Minecraft (might be a bit difficult though, as many people post 2000 are still in school)
+- No conclusion
```

Comment:
Could mention here that poeple have even written official sciency papers based on Factorio
https://www.reddit.com/r/factorio/comments/lgkcmy/the_factory_must_grow_automation_in_factorio/
https://arxiv.org/abs/2102.04871